src/js/index.js

```diff
@@ -76,29 +76,58 @@ class FHCookieGuard {
         });
     }
 
-    /**
-     * @private
-     */
     _onAcceptCookiesClick() {
-        this._setCookie(1);
+        this._setCookieConsentValue(this.options.cookieName, 1);
         this._enableCookieGuardedContent();
         this._closeCookieAlert();
     }
 
-    /**
-     * @private
-     */
     _onRefuseCookiesClick() {
-        this._setCookie(0);
+        this._setCookieConsentValue(this.options.cookieName, 0);
         this._closeCookieAlert();
     }
 
+    _initAutoAcceptCookieConsent() {
+        if (this._isCurrentPageExcluded() || !this.options.autoAcceptCookieConsentAfterRequestCount) {
+            return;
+        }
+
+        var { autoAcceptCookieConsentAfterRequestCount } = this.options;
+
+        if (!this._currentRequestCount) {
+            this._setRequestCounterSessionCookie(1);
+
+            return;
+        }
+
+        var currentRefreshCount = parseInt(this._currentRequestCount);
```

Comment:
overbodig als je het bij het ophalen van de cookie al doet zoals in mijn comment..